src/main/java/nl/tudelft/ewi/devhub/server/database/entities/issues/Issue.java

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.ewi.devhub.server.database.entities.issues;
+
+import java.net.URI;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import nl.tudelft.ewi.devhub.server.database.entities.User;
+
+
+@Data
+@Entity
+@Table(name="issues")
+@EqualsAndHashCode(callSuper = true)
+@NoArgsConstructor
+public class Issue extends AbstractIssue {
+
+	@NotNull	
+	@Column(name="title", length=128)
+	private String title;
+	
+	@NotNull
```

Comment:
I think these checks are only performed on persist time. They are done by the separate `hibernate-validator` anyway, and not part of the JPA core. (For that, you should use `@Column(nullable = false)`). But if we start storing the name, we should create one for the pull requests yes. That makes me question how the names are currently being generated. To answer your question, to our database migration, we simply add a query that sets the title field for pull requests, based on a sensible schema (`SET title = CONCAT("Pull Request", issueNumber, ...)`.